Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.8
+ENV PYTHONUNBUFFERED 1
```

Comment:
Can you explain why this with a comment?

Reply:
https://stackoverflow.com/questions/59812009/what-is-the-use-of-pythonunbuffered-in-docker-file . 

I'll add comments there